sdk/storage/Azure.Storage/src/Azure.Storage.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>12.0.0-preview.1</Version>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <GenerateAPIListing>false</GenerateAPIListing>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <NoWarn>$(NoWarn);NU5128</NoWarn>
+
+    <PackageTags>Storage;Blobs;$(PackageCommonTags)</PackageTags>
+    <Description>
+      Package that references the common Azure Storage libraries for things like blob manipulation.
```

Comment:
`Package that references all Microsoft Azure Storage libraries`?  They brand as `Microsoft Azure Storage` in our other packages and we'll probably forget to update this every time we add additional references.